src/utils/next.ts

```diff
@@ -26,7 +26,25 @@ type RegularProps = {
   [key: string]: any;
 };
 
+/**
+ * Defines the set of runtime environment variables available to [pages
+ * router](https://nextjs.org/docs/pages) code via
+ * [`getServerSideProps`](https://nextjs.org/docs/pages/building-your-application/data-fetching/get-server-side-props).
+ *
+ * Environment variables specified here must be included in the `ScaffoldedProps`
+ * object returned by `scaffold()`.
+ *
+ * @category Environment Variables
+ */
+export type ScaffoldedEnvVars = {
+  FEAT_AREAS: string | null;
+  MUIX_LICENSE_KEY: string | null;
+  ZETKIN_APP_DOMAIN: string | null;
+  ZETKIN_GEN2_ORGANIZE_URL: string | null;
+};
+
 export type ScaffoldedProps = RegularProps & {
+  envVars: ScaffoldedEnvVars;
```

Comment:
Wow I can't believe this was missed. I'm glad you caught it!

In ba86abc3c963a8f6b91cae4932382e69437d9878 (part of #2447) I made some minor changes and used `Environment['vars']` as the type. I don't particularly care about the name of the type, and I completely appreciate that using [indexed access types](https://www.typescriptlang.org/docs/handbook/2/indexed-access-types.html) is not everyone's preference, or perhaps even mine?

So if you want to change all of these places to use a shared, named type instead that is good IMO!